events/publish.go

```diff
@@ -50,7 +54,9 @@ func Publish(ctx context.Context, tmbus tmclient.EventsClient, name string, bus
 		return publishEvents(ctx, blkch, bus)
 	})
 
-	return g.Wait()
+	err = g.Wait()
+
+	return err
```

Comment:
don't need to `return err` if err declared with return type
can just do `return`